lib/dfe/analytics/services/entity_table_checks.rb

```diff
@@ -94,6 +94,8 @@ def entity_table_check_data(entity_name, order_column)
         end
 
         def send_entity_table_check_event(entity_name, entity_type, entity_tag, order_column)
+          return unless DfE::Analytics.enabled?
+
```

Comment:
Arguably this one isn't needed if we have a return in the entity_table_check_job

Reply:
This services gets called directly from the import rake tasks. I think this is still required, but is better moved to the call method. Or maybe we don't need at all if we put the guard in `import_entities.rake`